sdk/src/Services/SecurityToken/Custom/AmazonSecurityTokenServiceConfig.Extension.cs

```diff
@@ -100,6 +100,16 @@ public partial class AmazonSecurityTokenServiceConfig : ClientConfig
          };
 #endif
 
+        private static readonly ISet<string> legacyGlobalRegionSystemNames = new HashSet<string>();
```

Comment:
`ISet` doesn't extend back to .NET Framework 3.5

https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.iset-1?view=net-5.0#applies-to

This command failed during the dry-run:
```
  Custom\AmazonSecurityTokenServiceConfig.Extension.cs(103,33): error CS0246: The type or namespace name 'ISet<>' could not be found (are you missing a using directive or an assembly reference?) [D:\JenkinsWorkspaces\Stage-release-dry-run\AWSDotNetPublic\sdk\src\Services\SecurityToken\AWSSDK.SecurityToken.Net35.csproj] [D:\JenkinsWorkspaces\Stage-release-dry-run\AWSDotNetPublic\buildtools\build.proj]

  D:\JenkinsWorkspaces\Stage-release-dry-run\AWSDotNetPublic\buildtools\build.proj(172,9): error MSB3073: The command "dotnet build -c Release --no-incremental AWSSDK.Net35.sln /p:AWSKeyFile=c:\aws\AWSDotNet_Private.snk;SignAssembly=true;RunAnalyzersDuringBuild=true;RuleSetFileForBuild=" exited with code 1.
```

Reply:
I've revised it to follow the existing ifdef convention